tests/test_util_containers.py

```diff
@@ -9,47 +11,47 @@
 """
 class UtilTest(unittest.TestCase):
     def test_queue(self):
-        queue = util.Queue()
-        self.assertTrue(queue.isEmpty())
+        q = queue.Queue()
```

Comment:
Name the variable something consistent between the three, something like testQueue = queue.Queue(), testStack = ..., testPriorityQueue = ...